src/main/java/graphql/GraphQLContext.java

```diff
@@ -224,6 +224,13 @@ public static GraphQLContext of(Consumer<GraphQLContext.Builder> contextBuilderC
         return of(builder.map);
     }
 
+    /**
+     * @return a new and empty graphql context object
+     */
+    public static GraphQLContext getDefault() {
+        return GraphQLContext.newContext().build();
+    }
```

Comment:
This convenience method was added in a later version - backported to help with this PR